src/photosui.cs

```diff
@@ -472,9 +476,9 @@ interface PHProjectTypeDescriptionInvalidator
 	}
 
 	[iOS (8,0)]
+	[MacCatalyst (15,0)]
```

Comment:
web doc shows it's deprecated since iOS 13
Double check headers - it might not be decorated for catalyst (and since it's deprecated has little value to include)